api-flask/app/statsdata.txt

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Instead of these test files, let's create this sample data in a sample_requests.py file. You can simplify the alertdata a lot by just having a very simple GeoJSON, it doesn't need to be as big as the one you have there. It just needs to be a valid shape.